### DISCUSSION TITLE
"Unable to transform key"

### DISCUSSION THREAD

**Discussion by jficz** [ondat/trousseau#96] [Support]
Hi, got this [issue](https://github.com/ondat/trousseau/issues/91). It's blocking me from testing Trousseau with our cluster and I was unable to make progress. I'm not even sure it's a bug and I could use a tip. Any tip :)

The error message is generic and there is no useful log output from Trousseau so at this point I'm just guessing. Perhaps something with key rotation?

**Comment by romdalf** [comment]
Sorry @jficz - for some reasons, I am not receiving any alerting on discussions. 
Hopefully, we are currently working out the issue via the #91
